jupyter-extension/cypress/integration/repo-datum-mode-switch.ts

```diff
@@ -1,10 +1,11 @@
 describe('switching between repo and datum mode', () => {
   beforeEach(() => {
-    cy.unmountAllRepos();
     cy.resetApp();
     cy.isAppReady();
+    cy.unmountAllRepos();
     cy.openMountPlugin();
     cy.findAllByText('Mount');
+    cy.wait(3000);
```

Comment:
Kevin: Watching the video, one of our test suites reloaded halfway through the test, likely because not everything was ready when the test suite started. So this just waits before running the test to avoid any surprise mid-test reloads.